drivers/usb_c/tcpc/ucpd_stm32.c

```diff
@@ -1027,10 +1027,10 @@ static int ucpd_transmit_data(const struct device *dev,
 /**
  * @brief Tests if a received Power Delivery message is pending
  *
- * @retval true if message is pending, else false
+ * @retval 0 if there is no pending message
+ * @retval 1 if there is a pending message
  */
-static bool ucpd_is_rx_pending_msg(const struct device *dev,
-				   enum pd_packet_type *type)
+static int ucpd_is_rx_pending_msg(const struct device *dev, enum pd_packet_type *type)
 {
 	struct tcpc_data *data = dev->data;
 	bool ret;
```

Comment:
Non-blocking comment, this should be 
```suggestion
	bool int;
```
but you're deleting this function completely in a separate commit.

Reply:
Fixed that, thanks!